src/imarker_simple.cpp

```diff
@@ -42,21 +42,19 @@
 
 namespace rviz_visual_tools
 {
-IMarkerSimple::IMarkerSimple(const std::string& name, double scale) : nh_("~")
+IMarkerSimple::IMarkerSimple(const std::string& name, double scale, const geometry_msgs::Pose& initial_pose)
```

Comment:
Won't this break API for anyone using this? Would it be better add an additional constructor?